pkg/initializer_v2/test/e2e/test_model.py

```diff
@@ -0,0 +1,113 @@
+import os
+import runpy
+import shutil
+import tempfile
+
+import pytest
+
+import pkg.initializer_v2.utils.utils as utils
+from sdk.python.kubeflow.storage_initializer.constants import VOLUME_PATH_MODEL
+
+
+class TestModelE2E:
```

Comment:
@seanlaii @kubeflow/wg-training-leads @deepanker13 @Electronic-Waste @saileshd1402 What do you think about actually using Kubernetes to perform E2E tests for our initializer ?
E.g. we can deploy a single Pod that runs two initContainer for initializers and one Container to just verify that model and dataset exists under `/workspace/model` and `/workspace/dataset` dirs.

In that case, in our E2Es we verify that containers actually work for our initializers.

Do we see any values in such tests compare that we just run initializer Python scripts ?